### PR TITLE
Add Beneficiaries to tooltip of post footer

### LINKED ISSUE #1747: Display beneficiaries rewards on reward tooltip
We could show the rewards taken by beneficiaries feature directly on the reward tooltip like that:
image
This info is visible with the parameter `beneficiaries` on a post or comment:
https://api.steemjs.com/getContent?author=hyperfundit&permlink=150uy465

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1747 .

Changes:
- Add Beneficiaries to tooltip
screen shot 2018-04-13 at 7 29 18 pm

### CODE REVIEW

File: src/client/components/PayoutDetail.js [hunk 1]
```diff
@@ -31,6 +32,37 @@ AmountWithLabel.defaultProps = {
   amount: 0,
 };
 
+const getBeneficiariesPercent = user => {
+  try {
+    const weight = parseFloat(_.get(user, 'weight', 0)) / 10000;
+    // eslint-disable-next-line react/style-prop-object
+    return <FormattedNumber value={weight} style="percent" />;
+  } catch (error) {
```
Comment (Sekhmet) [inline src/client/components/PayoutDetail.js]: In what kind of scenarios is it possible to make this code throw an error?

Comment (jm90m) [reply src/client/components/PayoutDetail.js]: Thought a string would make it throw an error but all it does is make it NaN, so i could remove that and just have a NaN check

File: src/client/components/PayoutDetail.js [hunk 2]
```diff
@@ -31,6 +32,38 @@ AmountWithLabel.defaultProps = {
   amount: 0,
 };
 
+const getBeneficiariesPercent = user => {
+  const weight = parseFloat(_.get(user, 'weight', 0)) / 10000;
+
+  if (_.isNaN(weight)) {
```
Comment (Sekhmet) [inline src/client/components/PayoutDetail.js]: I think we could simplify it to:
```js
return <FormattedNumber value={_.isNan(weight) ? 0 : weight} style="percent" />;
```

**Comment by bonustrack** [comment]
@jm90m can you remove "Beneficiaries" and move the accounts after "Potential reward" line ?
image

**Comment by jm90m** [comment]
yeah, I did remove @bonustrack

**Pull request merged**